src/utility-types.ts

```diff
@@ -37,3 +38,21 @@ export interface MessageValue<
    */
   readonly parent?: MainWorker
 }
+
+/**
+ * TODO: Needs description.
```

Comment:
We need a description for whatever this is

Reply:
A wrapper containing a worker and two functions One to risolve and One to reject the promise.


Mmmm could be a good start